File: frontend/src/app/model/parameters/ReviewFilterParameters.ts
export class ReviewFilterParameters {
  commenterName?: string;
  comment?: string;
  fromRating?: number;
  toRating?: number;
  bookName?: string;

  constructor(commenterName?: string, comment?: string, fromRating?: number, toRating?: number, bookName?: string) {
    this.commenterName = commenterName;
    this.comment = comment;
    this.fromRating = fromRating;
    this.toRating = toRating;
    this.bookName = bookName;
  }
}

File: backend/src/main/java/com/softserve/utils/ReviewFilterParameters.java
package com.softserve.utils;

import lombok.Getter;
import lombok.Setter;

import java.math.BigInteger;

@Getter
@Setter
public class ReviewFilterParameters {
    private String commenterName;
    private String comment;
    private Integer fromRating;
    private Integer toRating;
    private String bookName;
    private BigInteger bookId;
}
